index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title></title>
+	<link href="https://fonts.googleapis.com/css?family=Comfortaa" rel="stylesheet">
+	<link rel="preconnect" href="https://fonts.googleapis.com">
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link href="https://fonts.googleapis.com/css2?family=Montserrat:ital@1&display=swap" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="main.css">
+</head>
+
+<body>
+	<div>
+		<h1 id="title">Survey Form Feedback</h1>
+		<p id="description">Please fill this form<sup>*</sup> to let us know how we can improve We Developed This Form
+		<h3>Lets fill the form</h3>
```

Comment:
I would suggest that Do not skip heading levels in HTML. It's difficult for web visitors using a screen reader to understand the contents of your web page when you skip heading levels.